PyInstaller/depend/bindepend.py

```diff
@@ -875,6 +876,36 @@ def get_python_library_path():
     # Applies only to non Windows platforms.
 
     if is_unix:
+
+        # Search if it is an Anaconda distribution
+        if re.findall('/anaconda', sys.executable):
+
+            # Relative anaconda path to the lib
+            rel_lib_path = 'lib/libpython3.7m.so.1.0'
```

Comment:
Ouch! This is bound to fail for Python 3.6 and 3.8 and if conda changes the compilation options (which will result in a different filename).

Also  os.path.join() should be used here

Reply:
Please I have used the regex to find a file that starts with 'libpython' and ends with 'm.so.1.0.'.
Making it difficult for any other file to make the cut.
```suggestion
            if re.findall('^libpython.*?m.so.1.0$', entry)
```